docker/configs/generators/generators/dot_properties.py

```diff
@@ -0,0 +1,47 @@
+import os
+import logging
+from . import *
+from cloudformation import get_stack_outputs
+
+LOGGER_NAME = os.path.splitext(os.path.basename(__file__))[0]
+LOGGER = logging.getLogger(LOGGER_NAME)
+OUTPUT_DIR = '/configs/etc/gu'
```

Comment:
This path is only correct within the context of docker. The generator scripts can be used outside of docker, so this should probably be configurable...